src/Makefile

```diff
@@ -1,52 +1,80 @@
+ifeq ($(shell which llvm-config-12),)
+  ifeq ($(shell which llvm-config-9),)
+    ifeq ($(shell which llvm-config),)
+      $(error Unable to locate the LLVM toolchain. \
+        Check your $$PATH or reinstall. Supported LLVM versions are 9 and 12)
+    else
+      LLVM_CONFIG = llvm-config
+      CLANG_FORMAT = clang-format
+    endif
+  else
+    LLVM_CONFIG = llvm-config-9
+    CLANG_FORMAT = clang-format-9
+  endif
+else
+  LLVM_CONFIG = llvm-config-12
+  CLANG_FORMAT = clang-format-12
+endif
+
+LLVM_VERSION := $(shell $(LLVM_CONFIG) --version | cut -d '.' -f 1)
+ifeq ($(LLVM_VERSION), 12)
+else ifeq ($(LLVM_VERSION), 9)
+  LLVM = -DLLVM9
```

Comment:
Can we name this variable LLVM_VRSN_FLAG instead of LLVM?

Reply:
Sure. Updated in b45282c.